parm/config/config.fv3.nco.static

```diff
@@ -19,21 +19,7 @@ case_in=$1
 
 echo "BEGIN: config.fv3"
 
-
-if [[ "$machine" = "WCOSS2" ]]; then
-   export npe_node_max=128
-elif [[ "$machine" = "WCOSS_DELL_P3" ]]; then
-   export npe_node_max=28
-elif [[ "$machine" = "WCOSS_C" ]]; then
-   export npe_node_max=24
-elif [[ "$machine" = "JET" ]]; then
-   export npe_node_max=24
-elif [[ "$machine" = "HERA" ]]; then
-   export npe_node_max=40
-elif [[ "$machine" = "ORION" ]]; then
-   export npe_node_max=40
-fi
-
+export npe_node_max=128
 
 # (Standard) Model resolution dependent variables
 case $case_in in
```

Comment:
Most of this can go too.

Reply:
I'll leave the other CASE blocks in config.fv3.nco.static for now, just in case. Will consider removing in the develop branch version. Thanks!